packages/notarization/storage.go

```diff
@@ -78,22 +78,6 @@ func (s *EpochCommitmentStorage) CachedECRecord(ei epoch.Index, computeIfAbsentC
 	return s.ecRecordStorage.Load(ei.Bytes())
 }
 
-func (s *EpochCommitmentStorage) SetFullEpochIndex(ei epoch.Index) error {
-	return s.setIndexFlag("fullEpochIndex", ei)
-}
-
-func (s *EpochCommitmentStorage) FullEpochIndex() (ei epoch.Index, err error) {
-	return s.getIndexFlag("fullEpochIndex")
-}
-
-func (s *EpochCommitmentStorage) SetDiffEpochIndex(ei epoch.Index) error {
-	return s.setIndexFlag("diffEpochIndex", ei)
-}
-
-func (s *EpochCommitmentStorage) DiffEpochIndex() (ei epoch.Index, err error) {
-	return s.getIndexFlag("diffEpochIndex")
-}
-
```

Comment:
These two were used to know at what index we have our full LedgerState and up to what index we have the diffs: I although forgot to set these upon `commitLedgerState`! That's why you removed them?

Reply:
I thought that they are not really used after loading the snapshot, so I removed full index and diff index. I assumed that when creating the snapshot we use lastCommittedEpochIndex and for diff we can simply retrieve it from parameter. We can add diff flag if it will be needed